app/server/testdata/workflow-valid-e2e-ghcr.yml

```diff
@@ -0,0 +1,69 @@
+name: scorecard-golang
+on:
+  workflow_dispatch:
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    - cron: '0 2 * * *'
+  push:
+    branches: [ main ]
+
+# Declare default permissions as read only.
+
+jobs:
+  scorecard-golang:
+    name: Scorecard Golang
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed to upload the results to code-scanning dashboard.
+      security-events: write
+      actions: read
+      contents: read
+      id-token: write # needed for keyless signing
+    strategy:
+      max-parallel: 2
+      fail-fast: false
+      matrix:
+        results_format: [sarif, json, default]
+        publish_results: [false, true]
+        include:
+          - results_format: sarif
+            upload_result: true
+          - results_format: json
+            upload_result: false
+          - results_format: default
+            upload_result: false
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2.4.0
+        with:
+          persist-credentials: false
+      - name: "Run analysis"
+        id: scorecard-run
+        uses: docker://ghcr.io/ossf/scorecard-action:latest
```

Comment:
this is the key difference between this and `app/server/testdata/workflow-valid-e2e.yml`